README.md

```diff
@@ -21,6 +21,13 @@ http://tedx.amsterdam/
 
 Clone the repository and run `npm install`:
 
+##Vagrent Installation
+Clone the repository and run vagrant up then vagrant ssh
```

Comment:
In documentation if you refer to code or commands please wrap them in backticks (`) for better readability e.g. `vagrant up` or `vagrant ssh`